docs/api-compatibility-matrix.md

```diff
@@ -0,0 +1,91 @@
+# API compatibility matrix
+
+## Epics
+
+Milestone 1 (DONE) - <https://scality.atlassian.net/browse/S3C-3768>
+
+Milestone 2 (IN PROGRESS) - <https://scality.atlassian.net/browse/S3C-3938>
+
+Milestone 3 (tickets TBD) - <https://scality.atlassian.net/browse/S3C-4027>
+
+## Object Storage Interoperability Service
+
+Reference API documentation - <https://code.vmware.com/apis/1034>
+
+Legend
+
+* `x` - implemented
+* `.` - not implemented
+* `o` - implemented with default stub
+* `*` - Mandatory APIs to integrate with OSE for essential functions, like basic S3 bucket/object CRUD operations
+
+| API  | Milestone 1 | Milestone 2 | Milestone 3 |
+|:-----|:------------|:------------|-------------|
+| headTenant * | . | . | . |
+| getTenant | . | . | . |
+| updateTenant * | . | . | . |
+| deleteTenant | . | . | . |
+| listTenants * | x | x | x |
+| createTenant * | x | x | x |
+| queryTenants * | x | x | x |
+| queryUsers * | . | x | x |
+| createUser * | . | x | x |
+| listUsers * | . | x | x |
+| getUserWithId * | . | x | x |
+| headUser | . | . | . |
+| updateUserStatus * | . | . | . |
+| deleteUser * | . | . | . |
+| getUserWithCanonicalID * | . | x | x |
+| queryCredentials * | . | . | . |
+| listCredentials * | . | x | x |
+| createCredential * | . | x | x |
+| getCredential * | . | . | . |
+| updateCredentialStatus | . | . | . |
+| deleteCredential | . | . | . |
+| getUsage | o | o | o |
+| getBucketList | . | . | . |
+| getBucketLoggingId | . | . | . |
+| getAnonymousUser | . | . | . |
+| getConsole | . | . | . |
+| getS3Capabilities * | o | x | x |
+| getInfo * | o | o | o |
+| refreshToken *  | . | . | . |
```

Comment:
This one came implemented with default stub and there needs no additional changes for this